src/schema/v2/home/fetch.ts

```diff
@@ -12,23 +12,33 @@ import {
 import blacklist from "lib/artist_blacklist"
 import { LoadersWithAuthentication } from "lib/loaders/loaders_with_authentication"
 import { LoadersWithoutAuthentication } from "lib/loaders/loaders_without_authentication"
+import moment from "moment"
 
 export const featuredFair = (
   fairsLoader: LoadersWithoutAuthentication["fairsLoader"]
 ) => {
   return fairsLoader({
     size: 5,
     active: true,
+    sort: "-start_at",
```

Comment:
FYI, I believe the fairs API endpoint also supports a `status` param which you might be able to use in order to filter down to running fairs without the explicit datetime comparison here.

Reply:
You're right. The fair endpoint supports a status param. Thanks!